kube/service.yml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: little-help-book
+
+  namespace: little-help-book
+  annotations:
+    getambassador.io/config: |
+      ---
+      apiVersion: ambassador/v0
+      kind: Mapping
+      name: stage-little-help-book-mapping
+      host: mvpstudio.stage.apps.mvpstudio.org
+      prefix: /api
+      rewrite: /api
+      service: mvpstudio-web-api.stage-mvpstudio-web
+      ---
+      apiVersion: ambassador/v0
```

Comment:
As above, URLs and service aren't right.

Reply:
PR #3